### PR TITLE
rm unnecessary fs.Manager

### PR DESCRIPTION
**Pull request by xiaoanyunfei**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

> /kind cleanup

**What this PR does / why we need it**:
I just wonder why create a new fs.Manager instead of the default cont.manager
 They have exactly the same parameters, and nothing has changed
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
 None
```
/sig node

### CODE REVIEW

File: pkg/kubelet/cm/container_manager_linux.go
Comment (mattjmcnaughton) [inline pkg/kubelet/cm/container_manager_linux.go]: From looking at https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/container_manager_linux.go#L338, I believe you are correct that this `fs.Manager` is identical to the other `cont.Manager` which we already created.

**Review by mattjmcnaughton — COMMENTED** [review]
/ok-to-test

Overall this makes sense to me. Please ping me again after the test suite runs and I will approve/lgtm if all passes.

One note... I'm not sure that I would call the manager you are deleting `unused`. Because it actually is used by `const.ensureStateFunc`. I just think that I would call it unnecessary. Mind updating your commit/pr message accordingly?

In addition, please replace your current comment for `Does this PR introduce a user facing change` with `NONE`. That section is intended for release notes, and I don't believe we need a release note for this change.

**Comment by xiaoanyunfei** [comment]
Hi @mattjmcnaughton , I have change the git commit and pr message, please review again

**Review by tallclair — COMMENTED** [review]
This lgtm, but if anything modifies the containers manager it would behave differently (but probably correctly?).

/assign @vishh 
Incase I'm missing any historical context.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Agree with @tallclair that this change gives us the greatest probability of the code continuing to work correctly in the event of a change.

**Comment by xiaoanyunfei** [comment]
ping @resouer

**Pull request closed without merging**
